pkg/controller/controller.go

```diff
@@ -358,7 +371,15 @@ func (c *controller) getServiceClassPlanAndServiceBrokerForServiceInstanceCreden
 }
 
 // ServiceBroker utility methods - move?
+// getServiceClassPlanAndBrokerForBinding is a sequence of operations that's
+// done to validate service plan, service class exist, and handles creating
+// a brokerclient to use for a given Instance.
+func (c *controller) getServicePlan(name string) (*v1alpha1.ServicePlan, error) {
+	// use the ref to get the real plan
+	return c.servicePlanLister.Get(name)
```

Comment:
Note - at some point this will need to be scoped to just the service.  But its all based on how we deal with names of plans, etc....

Reply:
Not sure I agree with the scope of that. This does exactly and only what it should. TODO should be an issue, maybe?